resume.md

```diff
@@ -1,49 +1,47 @@
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
-
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
-
-## PROJECTS
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Developed PROJECT NAME in order to FEATURE solving PROBLEM for POPULATION
-* Launched PROJECT NAME to the Apple App Store resulting in X Downloads and Y Monthly Users
-* Beta Tested to X Number of users and collected Crash Data using analytics LIBRARY
-* ...
-
-**Project Name**: *Brief One Liner TL;DR* | tinyurl ... *Month 20XX - Month 20XX*
-* Contributed to OPEN SOURCE LIBRARY, a popular module for FEATURE, by closing X tasks and merging Y Pull Requests.
-* Created Demo Tutorial series to demonstrate FEATURE for early adopters prior to production push.
-* Managed App Version release cycles to maintain proper software versioning for major and minor app increments.
-* ...
-
-## AWARDS/ACKNOWLEDGEMENTS
-* **COLLEGE Alumni Scholarship Recipient** - Alumni Scholarship Committee... *Month 20XX*
-* **1st Place Best Hack** - Hackathon Event... *Month 20XX*
-
-## Events/Conferences
-* **Cal Hacks 7.0 (remote)** - Berkeley, CA ... *Month 20XX*
-* **Cal Hacks 6.0** - Berkeley, CA ... *Month 20XX*
-* **WWDC 2019** - San Jose, CA ... *Month 20XX*
+**CS 128 Course Assistant**: *September 2020 - Present*
+* Course assistant for CS 128, a new core Computer Science class to be implemented Spring 2021
+* Developed engaging assignments in C++ for future students
+* Worked cohesively with other assistants to produce the course material
+
+**Sandia National Laboratories Space Blimp Challenge**: *November 2019*
+* Led team that developed new software that improves the management of experimental flight data 
+* Effectively utilized Java to parse through given data for maximum time efficiency
+
+**Alameda County and Engineering Fair**: *April 2018*
+* Presented a research study focused on Artificial Intelligence technologies and Machine Learning to industry professionals
+* Analyzed Artificial Intelligence services such as Amazon Rekognition Services, Google Cloud Vision Service, Microsoft’s Cognitive Services Vision and Face on their image correctness, accuracy, and comprehensiveness 
+
+## LEADERSHIP/ACTIVITIES
+**Illinois Formula Electric**: *August 2020 - Present*
+* Member of the Data Acquisition Analysis (DAQA) subsystem
+* Worked collaboratively with clubmates to enhance data visualization by using Python
```

Comment:
Be more specific, did you use a specific library/module to accomplish this?  How much did you improve?  Did you create graphs / diagrams?  Anything online that we can see?